app/views/courses/TeacherClassesView.coffee

```diff
@@ -160,7 +160,7 @@ module.exports = class TeacherClassesView extends RootView
       @supermodel.trackRequest req
 
     # TODO: Any reference to paidTeacher can be cleaned up post Teacher Appreciation week (after 2019-05-03)
-    @paidTeacher = me.isAdmin() or me.isTeacher() and /@codeninjas.com$/i.test me.get('email')
+    @paidTeacher = me.isAdmin() or me.isCreatedByClient() or me.isTeacher() and /@codeninjas.com$/i.test me.get('email')
```

Comment:
I think this should be `me.isAdmin() or me.isTeacher() and (/@codeninjas.com$/i.test me.get('email') or me.isCreatedByClient())` i.e. `isCreatedByClient` condition can also be combined with `isTeacher`.

Reply:
Same for all three files where this has been changed.